docs/guides/volume-populator/overview/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Volume Populator Overview | KubeStash
+description: An overview of how Volume Populator works in KubeStash
+menu:
+  docs_{{ .version }}:
+    identifier: volume-populator-overview
+    name: How Volume Populator works?
+    parent: volume-populator
+    weight: 10
+product_name: kubestash
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+# Populate Volume Using KubeStash
+
+This guide will give you an overview of how volume populator process works in KubeStash.
+
+## How Volume Populator Process Works?
+
+The following diagram shows how KubeStash restores backup data into a specific volume and attaching it to user-provided Persistent Volume Claims (PVCs) for usage of various workload applications.
+
```

Comment:
The following diagram shows how KubeStash populate a Persistent Volume Claim with data sourced from a KubeStash Snapshot.